blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfigStatePreloader.java

```diff
@@ -83,4 +88,22 @@ public String getStateJson() {
     private String getBlueOceanPluginVersion(){
         return Jenkins.getInstance().getPlugin("blueocean-web").getWrapper().getVersion();
     }
+
+    /**
+     * Returns true if the orgatizations are enabled on this instance
+     * 
+     * @return true if the organizations are enabled
+     */
+    public static boolean isOrganizationsEnabled() {
+        return organizationsEnabled;
+    }
+
+    /**
+     * Sets the organizations enabled/disabled flag
+     * 
+     * @param organizationsEnabled true to enable the organizations
+     */
+    public static void setOrganizationsEnabled(boolean organizationsEnabled) {
```

Comment:
Don't think such static setter on this class is good idea. Why not go with extension point approach, something like:

```java
public abstract class BlueOceanConfiguration implements ExtensionPoint {
    public abstract boolean isOrganizationsEnabled();
}
```
This way we can add more configuration elements as needed instead of adding methods around? 🐛